src/views/LiquidityPool/components/EarnByStakingInfoBox.tsx

```diff
@@ -38,6 +38,7 @@ export function EarnByStakingInfoBox({
     : data?.apyData.baseRewardsApy;
 
   const textColor = selectedPoolAction === "add" ? "aqua" : "warning";
+  const isStakingPoolEnabled = data?.poolEnabled;
```

Comment:
Nit: we may want to convert this into a boolean for the case that `data === undefined`.
```suggestion
  const isStakingPoolEnabled = !!data?.poolEnabled;
```

Reply:
Yes good point